isaric/parsers/ccpuk_observations.toml

```diff
@@ -0,0 +1,465 @@
+[adtl.defs.admissionDateHierarchy]
+combinedType = "firstNonNull"
+        fields = [
+            { field = "hostdat", description = "Admission date at this facility" },
+            { field = "cestdat", description = "Onset date of first/earliest symptom" },
+            { field = "dsstdat", description = "Date of enrolment" },
+            { field = "daily_dsstdat", description = "Date of assessment" }
+        ]
+
+[adtl.defs.followupDateHierarchy]
+combinedType = "firstNonNull"
+fields = [
+{ field = "flw_survey_date", description = "About you and your COVID-19 illness (if you're completing this survey on behalf of a child or adult that you care for, all the questions relate to their health and wellbeing)" },
+{ field = "dsstdtc", description = "Outcome date", if = { dsterm = 4 } },
+{ field = "dsstdtc_v2", description = "Outcome date", if = { dsterm_v2 = 4 } }
+]
+
+[[observation]]
+name = "haemorrhage"
+date = { ref = "observationDateHierarchy" }
+phase = "admission"
+is_present = { field = "cough_ceoccur_v2", ref = "Y/N/NK" }
+if = { bleed_ceoccur_v2 = { "!=" = 3} }
+#description = "Bleeding (Haemorrhage)"
+
+[[observation]]
+name = "cough"
+date = { ref = "observationDateHierarchy" }
+phase = "admission"
+is_present = { if field = "cough_ceoccur_v2", ref = "Y/N/NK" }
+if = { cough_ceoccur_v2 = { "!=" = 3} }
+#description = "Cough"
+
+[[observation]]
+name = "cough"
+date = { ref = "followupDateHierarchy" }
+phase = "followup"
+is_present = { if field = "flw_cough", values = { 1 = true, 0 = false } }
+if = { flw_cough = { "!=" = 99} }
+#description = "Cough"
+
+[[observation]]
+name = "avpu"
+date = { field = "daily_dsstdat", description = "Date of assessment" }
+phase = "study"
+value_char = { field = "avpu_vsorres", values = { 1 = "Alert", 2 = "Verbal", 3 = "Pain", 4 = "Unresponsive" }}
+if = { avpu_vsorres = { "!=" = 5} }
+
+[[observation]]
+name = "oxygen_saturation_percent"
+phase = "admission"
+date = { field = "hostdat", description = "Admission date at this facility" } # USE one of the defs above?
+context = { field = "oxy_vsorresu", description = "Oxygen saturation on:", values = { 1 = ["Room air"], 2 = ["Oxygen therapy"]} }
+value_num = { field = "oxy_vsorres" }
+if = { oxy_vsyn = 1 }
+
+[[observation]]
+name = "glasgow_coma_score"
+phase = "study"
+date = { field = "daily_dsstdat", description = "Date of assessment" }
+value_num = { field = "daily_gcs_vsorres" }
+if = { daily_gcs_lbyn = 1 }
+#description = "Glasgow Coma Score (GCS/15)"
+
+[[observation]]
+name = "conjunctivitis"
+phase = "admission"
+date = { ref = admissionDateHierarchy }
```

Comment:
admissionDateHierarchy should be quoted

Reply:
I have changed this - should come through in the new PR, hopefully.